pkg/docker/container.go

```diff
@@ -72,7 +72,7 @@ func EnsureContainer(ctx context.Context, log *zap.SugaredLogger, cli *client.Cl
 
 		err := cli.ContainerStart(ctx, ci.ID, types.ContainerStartOptions{})
 		if err != nil {
-			log.Errorw("starting container failed", "container_id", container.ID)
+			log.Errorf("starting container failed", "container_id %s: %v", opts.ContainerName, err)
```

Comment:
I don't think this will have the right effect. I'd suggest you keep `Errorw` and add an `"error", err` tuple.